<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/pic/KTLT(1).png);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblHoTen">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Họ và Tên:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSdt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Số điện thoại:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCCCD">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>CCCD/CMT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTensach">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Tên sách:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMasach">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Mã sách:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDiachi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Địa chỉ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledHovaTen">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledSdt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledCCCD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTensach">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledMasach">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledDiachi">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ptnxacnhan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>279</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 5px;
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Xác nhận</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ptnquaylai">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 5px;
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Quay lại</string>
    </property>
   </widget>
   <zorder>lblSdt</zorder>
   <zorder>lblCCCD</zorder>
   <zorder>lblTensach</zorder>
   <zorder>lblMasach</zorder>
   <zorder>lblDiachi</zorder>
   <zorder>ledHovaTen</zorder>
   <zorder>ledSdt</zorder>
   <zorder>ledCCCD</zorder>
   <zorder>ledTensach</zorder>
   <zorder>ledMasach</zorder>
   <zorder>ledDiachi</zorder>
   <zorder>lblHoTen</zorder>
   <zorder>ptnxacnhan</zorder>
   <zorder>ptnquaylai</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="giaodien1.qrc"/>
 </resources>
 <connections/>
</ui>
